content/en/docs/ops/configuration/traffic-management/protocol-selection/index.md

```diff
@@ -12,11 +12,36 @@ owner: istio/wg-networking-maintainers
 test: no
 ---
 
-Istio supports proxying all TCP traffic by default, but in order to provide additional capabilities,
-such as routing and rich metrics, the protocol must be determined.
-This can be done automatically or explicitly specified.
+Istio supports proxying any TCP traffic. This includes HTTP, HTTPS, gRPC, as well as raw TCP protocols.
+In order to provide additional capabilities, such as routing and rich metrics, the protocol must be determined. This can be done automatically or explicitly specified.
 
-## Manual protocol selection
+Non-TCP based protocols, such as UDP, are not proxied. These protocols will continue to function as normal, without
+any interception by the Istio proxy but cannot be used in proxy-only components such as ingress or egress gateways.
+
+## Automatic protocol selection
+
+Istio can automatically detect HTTP and HTTP/2 traffic. If the protocol cannot automatically be determined, traffic will be treated as plain TCP traffic.
+
+### Server first protocols
+
+Some protocols are "Server First" protocols, which means the server will send the first bytes. In these cases, automatic protocol detection
+will not function correctly, as Istio will determine the protocol based on the initial bytes. If you are using a server first protocol, you should
+follow the [explicit protocol selection instructions](#explicit-protocol-selection) to declare the protocol.
+
+The following protocols are known to be incompatible with automatic protocol selection, and must also be explicitly declared or they will be treated as TCP:
+
+|Protocol|Port|
+|--------|----|
+| SMTP   |25  |
+| DNS    |53  |
+| MySQL  |3306|
+| MongoDB|27017|
+
+{{< tip >}}
+Since TLS communication is not server first, any TLS based traffic will support protocol detection, even if it is wrapping a protocol that does not support detection, such as MySQL.
+{{< /tip >}}
```

Comment:
One more nit. The incompatible protocols part looks like it's in the `Server first protocols` subsection, so I suggest either removing the section heading or reordering the text like this:
```suggestion
The following protocols are known to be incompatible with automatic protocol selection, and must be
[explicitly declared](#explicit-protocol-selection) or they will be treated as TCP:

|Protocol|Port|
|--------|----|
| SMTP   |25  |
| DNS    |53  |
| MySQL  |3306|
| MongoDB|27017|

{{< tip >}}
Since TLS communication is not server first, any TLS based traffic will support protocol detection, even if it is wrapping a protocol that does not support detection, such as MySQL.
{{< /tip >}}

### Server first protocols

Some protocols are "Server First" protocols, which means the server will send the first bytes. In these cases, automatic protocol detection
will not function correctly, as Istio will determine the protocol based on the initial bytes. If you are using a server first protocol, you should
follow the explicit protocol selection instructions, below, to declare the protocol.
```